### DISCUSSION TITLE
Welcome to Foundry Discussions!

### DISCUSSION THREAD

**Discussion by gakonst** [foundry-rs/foundry#112] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by transmissions11** [comment]
gm

**Comment by ghost** [comment]
: ]

**Comment by jparklev** [comment]
gm ~ V excited at the idea of inc compilations. Don't have too much time atm, but will certainly contribute where I can

**Reply by gakonst** [reply]
Nice! @mattsse has been revamping the solc bindings in ethers here: https://github.com/gakonst/ethers-rs/pull/522.

We could also pull in the solc building pipeline from this crate there and have a full fledged build system as a crate!

**Reply by jparklev** [reply]
🥂 was just reading it over – looks great! Also I saw solidity just closed out this issue https://github.com/ethereum/solidity/issues/11756 that so we can get absolute, resolved (post remapping) paths for imports which I think will help for this as well!(?)

**Comment by pmerkleplant** [comment]
gm frens.
I'm a solidity auditor who fell in love with the foundry toolchain. 
Starting to learn Rust now (finally) and looking forwards to being able to contribute 🤞
